packages/icons-workflow/README.md

```diff
@@ -0,0 +1,106 @@
+## Description
+
+Spectrum Workflow Icons delivered in a mutable template tag so that they can be leveraged across various frameworks. The default export of this package pre-applies the `html` template tag from `lit-html` for ease of use in the Spectrum Web Components library. Continue reading below for information on using these icons in projects powered by other frameworks.
```

Comment:
I'd link out to https://spectrum.adobe.com/page/icons/ in this description -- I'd expect folks new to spectrum to be confused otherwise.